### PR TITLE
Alternative extracting sub-html from a[title] and img[alt] attributes

### PR DESCRIPTION
**Pull request by dkrnl**
Hi Sachin!
Thank you.

### CODE REVIEW

**Comment by abnersajr** [comment]
Thanks man, I will accept this soon and make the addition on docs.

File: src/js/lightgallery.js [hunk 1]
```diff
@@ -460,6 +460,12 @@
                 subHtmlUrl = this.$items.eq(index).attr('data-sub-html-url');
             } else {
                 subHtml = this.$items.eq(index).attr('data-sub-html');
+                if (!subHtml) {
```
Comment (sachinchoolur) [inline src/js/lightgallery.js]: Hi Guys,
Can we simplify the code?
``` javascript 
subHtml = this.$items.eq(index).attr('data-sub-html') || this.$items.eq(index).attr('title') || this.$items.eq(index).find('img').attr('alt')
```
And how do we manage if there are multiple images under a selector?

File: src/js/lightgallery.js [hunk 2]
```diff
@@ -469,7 +475,7 @@
                 // get first letter of subhtml
                 // if first letter starts with . or # get the html form the jQuery object
                 var fL = subHtml.substring(0, 1);
-                if (fL === '.' || fL === '#') {
+                if (subHtmlSelector && fL === '.' || fL === '#') {
```
Comment (dkrnl) [inline src/js/lightgallery.js]: Flag `subHtmlSelector` protect possible jQuery-injection from WYSIWYG-editor:
```html
<a href="full.jpeg"><img src="thumb.jpeg" alt="#This not query"></a>
```

File: src/js/lightgallery.js [hunk 3]
```diff
@@ -460,6 +460,12 @@
                 subHtmlUrl = this.$items.eq(index).attr('data-sub-html-url');
             } else {
                 subHtml = this.$items.eq(index).attr('data-sub-html');
+                if (!subHtml) {
+                    subHtml = this.$items.eq(index).attr('title') || this.$items.eq(index).find('img').attr('alt');
```
Comment (dkrnl) [inline src/js/lightgallery.js]: > And how do we manage if there are multiple images under a selector?
jQuery get first attribute element from collection. And multiple thumb-images its possible?

**Comment by sachinchoolur** [comment]
Hi @dkrnl ,
I have included this feature in v1.2.18. Also added an option to disable/enable alt/title support in LG settings. 
Thank you very much for your contribution..

**Pull request closed without merging**